packages/x-charts/src/ChartContainer/useChartContainerProps.ts

```diff
@@ -32,52 +31,43 @@ export const useChartContainerProps = (
     plugins,
     sx,
     title,
-    viewBox,
     xAxis,
     yAxis,
     zAxis,
     skipAnimation,
     ...other
   } = props;
 
-  const {
-    containerRef,
-    width: dWidth,
-    height: dHeight,
-  } = useChartContainerDimensions(width, height, resolveSizeBeforeRender);
+  const resizableContainerProps = other;
```

Comment:
Does it still have a purpose? The props type is defined as follow

```
interface ChartContainerProps extends ChartDataProviderProps, ChartsSurfaceProps {}
```

Then `other` should be empty